gemrb/plugins/SDLVideo/SDLVideo.cpp

```diff
@@ -22,21 +22,21 @@
 
 #include "Interface.h"
 #include "Video/RLE.h"
+#include "SDLInitSubSystemWrapper.h"
 #include "SDLPixelIterator.h"
 
 using namespace GemRB;
 
 SDLVideoDriver::~SDLVideoDriver(void)
```

Comment:
you should remove the destructor altogether.

Reply:
thanks, done.